cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/batch/DynamicPartitionerWithAvroTest.java

```diff
@@ -119,6 +128,19 @@ public void apply() {
         }
       });
 
+    final PartitionedFileSet pfs = datasetCache.getDataset(OUTPUT_DATASET);
+
+    Transactions.createTransactionExecutor(txExecutorFactory, (TransactionAware) pfs).execute(
+      new TransactionExecutor.Subroutine() {
+        @Override
+        public void apply() throws IOException {
+          writeFile(pfs, PartitionKey.builder().addLongField("time", now).addIntField("zip", 95111).build());
+          writeFile(pfs, PartitionKey.builder().addLongField("time", now).addIntField("zip", 98123).build());
+          writeFile(pfs, PartitionKey.builder().addLongField("time", now).addIntField("zip", 84125).build());
+          // org.apache.hadoop.mapred.FileAlreadyExistsException: Final output path
```

Comment:
Ah, this was just a note for me, as I was modifying the test case. I'll remove it.